src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties

```
@@ -213,7 +213,8 @@ OPF_094=The "media-type" attribute is required for "%1$s" links.
 OPF_095=The "media-type" attribute of "voicing" links must be an audio MIME type, but found "%1$s".
 OPF_096=Non-linear content must be reachable, but found no hyperlink to "%1$s".
 OPF_096b=No hyperlink was found to non-linear document "%1$s", please check that it can be reached from scripted content.
-OPF_097=Resource "%1$s" is listed in the manifest, but no reference to it was found in content documents. 
+OPF_097=Resource "%1$s" is listed in the manifest, but no reference to it was found in content documents.
+OPF_098=The "href" attribute must not reference resources via elements in the package document itself, but found URL "%1$s".
```

Minor nit, but could this be explained as a must:

```suggestion
OPF_098=The "href" attribute must reference resources, not elements in the package document, but found URL "%1$s".
```

---

Yeah, I copied the language from the spec 🙂, but I agree your suggested wording makes more sense.

At some point it would be nice to have the time to go over all the messages and try to do some copyediting to harmonize the language and tone… but well, that's for another version!